provisioning/transport/amqp/package.json

```diff
@@ -35,7 +35,7 @@
     "alltest": "istanbul cover ../../../node_modules/mocha/bin/_mocha -- --reporter spec test/_*_test*.js",
     "ci": "npm -s run lint && npm -s run build && npm -s run alltest-min && npm -s run check-cover",
     "test": "npm -s run lint && npm -s run build && npm -s run unittest",
-    "check-cover": "istanbul check-coverage --statements 93 --branches 81 --functions 87 --lines 93"
+    "check-cover": "istanbul check-coverage --statements 95 --branches 88 --functions 90 --lines 96"
   },
```

Comment:
nice!!!!

Reply:
One tries.

---
In reply to: [282194865](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282194865) [](ancestors = 282194865)